pkg/apis/v1/nodeclaim_validation.go

```diff
@@ -68,74 +67,75 @@ func (in *NodeClaim) SupportedVerbs() []admissionregistrationv1.OperationType {
 }
 
 // Validate the NodeClaim
-func (in *NodeClaim) Validate(_ context.Context) (errs *apis.FieldError) {
-	return errs.Also(
-		apis.ValidateObjectMetadata(in).ViaField("metadata"),
-		in.Spec.validate().ViaField("spec"),
-	)
+func (in *NodeClaim) Validate(_ context.Context) error {
```

Comment:
Since, we don't do any runtime validation on the NodeClaim, do we still need this validations?

Reply:
Good point, do we need it and if so should it be added in this PR?

Reply:
So does that mean that all of the tests in validate are also no longer valid or useful until the CEL validation is added?

Reply:
So the CEL validation tests are currently in the package. Yeah, I don't think we need this validation or webhook tests anymore since they were only used for Webhook validation